components/eventing-controller/pkg/deployment/publisher_deployment.go

```diff
@@ -282,6 +283,9 @@ func getBEBEnvVars(publisherConfig env.PublisherConfig) []v1.EnvVar {
 }
 
 func getNATSEnvVars(publisherConfig env.PublisherConfig) []v1.EnvVar {
+	// reading the NATS backend config in order to get Jetstream related env variables
+	natsConfig := env.GetNatsConfig(10, 3*time.Second)
```

Comment:
the `10` and `*time.Second` are default values, and they stay for `maxReconnects` and `reconnectWait` which are not relevant for epp, so we can go with these default. values.